src/OSPSuite.Presentation/Presenters/Importer/IModalImporterPresenter.cs

```diff
@@ -106,7 +109,12 @@ ImporterConfiguration configuration
             results.Each(r => r.ConfigurationId = configurationId);
          }
 
-         _importerPresenter.OnTriggerImport += (s, d) => { _view.CloseOnImport(); };
+         _importerPresenter.OnTriggerImport += (s, d) => 
+         {
+            _shouldPromtBeforeClose = false;
+            _view.CloseOnImport(); 
```

Comment:
Oh see. This methods already exists and it is called CloseView. 
From the name it looked it was triggering some event.. (Onxxx) .Naming is really everything in programming :)

Reply:
> "There are only two hard things in Computer Science: cache invalidation and naming things"?
> Phil Karlton

😅